src/libstd/collections/hash/map.rs

```diff
@@ -2161,6 +2161,37 @@ impl<'a, K, V> OccupiedEntry<'a, K, V> {
     fn take_key(&mut self) -> Option<K> {
         self.key.take()
     }
+
+    /// Replaces the entry, returning the old key and value.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #![feature(map_entry_replace)]
```

Comment:
I think we include the feature toggle in the docs so that end users know that it is necessary. So I'd remove the `# ` prefix here.